site/src/components/Margins/Margins.tsx

```diff
@@ -13,8 +13,15 @@ const widthBySize: Record<Size, number> = {
   small: containerWidth / 3,
 };
 
-export const Margins: FC<JSX.IntrinsicElements["div"] & { size?: Size }> = ({
+type MarginsProps = JSX.IntrinsicElements["div"] & {
+  size?: Size;
+  verticalMargin?: string | number;
+};
+
+export const Margins: FC<MarginsProps> = ({
   size = "regular",
+  verticalMargin = 0,
+  children,
```

Comment:
Maybe, would be better to extend the HTMLDivElement props and let the parent set the styles. IMO it is better for extensibility.